vars/mavenCanaryRelease.groovy

```diff
@@ -155,9 +159,10 @@ def zeroFill(v, count) {
  * an array of strings of equal length. e.g.
  * "1.2", "3.4.5" returns [1, 2, 0], [3, 4, 5]
  * "1.2.3-5", "3.4.5" returns [1, 2, 3, 15], [3, 4, 5]
+ * "1.2.3-5-rhor", "3.4.5" returns [1, 2, 3, 15], [3, 4, 5]
```

Comment:
1,2, 3, **5** instead of **15**

Reply:
and IIUC, `Integer.valueOf('rhor') ` return 0 - right?

Reply:
fixed